test/integration/node-specific/abort_signal.test.ts

```diff
@@ -942,7 +942,8 @@ describe('AbortSignal support', () => {
       const start = performance.now();
       const result = await collection.findOne({}, { signal }).catch(error => error);
       const end = performance.now();
-      expect(end - start).to.be.lessThan(10); // shouldn't wait for the blocked connection
+      // TODO(NODE-6833): This duration was bumped from 10 to 40 to reduce flakiness, if this fails again investigate it.
```

Comment:
```suggestion
      // This duration was bumped from 10 to 40 to reduce flakiness, if this fails again investigate it.
```
There is no TODO anymore.

Reply:
I thought we still wanted to link the ticket if it flakes again we know not to bump it again